torch/fx/experimental/proxy_tensor.py

```diff
@@ -887,20 +889,16 @@ def get_innermost_proxy_mode():
 
 
 @contextlib.contextmanager
-def disable_proxy_modes_tracing(enable_current=False):
```

Comment:
Hmm :( you might need to co-land this with internal? https://www.internalfb.com/code/fbsource/[fc522656777004802315d528e8db22dee53a5cac]/fbcode/sigmoid/frontend/aot_export.py?lines=675

Reply:
I will delete the internal usage first: D53723788